src/pybert/pybert.py

```diff
@@ -645,7 +638,7 @@ def __init__(self, run_simulation=True, gui=True):
 
         # Super-class initialization is ABSOLUTELY NECESSARY, in order
         # to get all the Traits/UI machinery setup correctly.
-        super(PyBERT, self).__init__()
+        super().__init__()
```

Comment:
Same question as above.

Reply:
Same story here, that is no longer required as of python 3.0!  https://docs.python.org/3/library/functions.html#super